new file:   assets/images/desktop/logo-handset.svg

ulia454012 opened the pull request:
new file:   assets/images/desktop/logo-instagram.svg
        new file:   assets/images/desktop/logo-telegram.svg
        new file:   assets/images/desktop/logo-whatsapp.svg
        modified:   assets/styles/css/styles.css
        modified:   assets/styles/sass/fonts.sass
        modified:   assets/styles/sass/mixin.sass
        modified:   assets/styles/sass/styles.sass
        new file:   assets/styles/sass/variables.sass
        modified:   index.html

ulia454012 committed:
new file:   assets/images/desktop/logo-handset.svg
        new file:   assets/images/desktop/logo-instagram.svg
        new file:   assets/images/desktop/logo-telegram.svg
        new file:   assets/images/desktop/logo-whatsapp.svg
        modified:   assets/styles/css/styles.css
        modified:   assets/styles/sass/fonts.sass
        modified:   assets/styles/sass/mixin.sass
        modified:   assets/styles/sass/styles.sass
        new file:   assets/styles/sass/variables.sass
        modified:   index.html

taorkon committed:
Delete variables.sass

taorkon committed:
Update mixin.sass

taorkon committed:
Update styles.sass

taorkon reviewed (changes requested):
- Поменяй пожалуйста запись элементов на &__ (см как сделаны другие блоки)
- Миксины h2-7 мы убираем, они будут в шрифтах 
- Пока можно просто прописать шрифты в общих стилях
- Надпись про инсту не надо в хедере
- Названия классов длинноваты, можешь поменять на то что сейчас в футере в осн ветке

taorkon committed:
review

ulia454012 committed:
modified:   assets/styles/css/styles.css
        modified:   assets/styles/sass/colors.sass
        modified:   assets/styles/sass/styles.sass
        deleted:    assets/styles/sass/variables.sass
        modified:   index.html

ulia454012 committed:
Merge branch 'dev-Yulia' of https://github.com/taorkon/english-school into dev-Yulia

ulia454012 committed:
new file:   assets/fonts/MontserratAlternates-Regular.woff
        deleted:    assets/fonts/dummy.txt
        deleted:    assets/images/desktop/dummy.txt
        new file:   assets/images/desktop/logo-english-school.svg
        deleted:    assets/images/desktop/shtamp.png
        modified:   assets/styles/css/styles.css
        modified:   assets/styles/sass/fonts.sass
        modified:   assets/styles/sass/styles.sass
        modified:   index.html

taorkon commented:
1. Поменяй пожалуйста названия классов как в футере в осн ветке
2. Фон отъезжает при сжатии окна, а иконки выезжают за него(лучше это решать сразу с адаптивом), см скрин
3. Классы в навигации не по бем header__menu-item либо header__item будет корректно

![Screenshot from 2023-04-23 23-33-46](https://user-images.githubusercontent.com/27294801/233865116-c77bb640-7d31-42fb-84c7-bb9cd1a987b0.png)

taorkon reviewed (changes requested):
см коммент тут https://github.com/taorkon/english-school/pull/5#issuecomment-1519151554

ulia454012 committed:
modified:   assets/styles/css/styles.css
        modified:   assets/styles/sass/fonts.sass
        modified:   assets/styles/sass/styles.sass
        modified:   index.html

ulia454012 committed:
Add files via upload

ulia454012 committed:
Update README.md

taorkon committed:
header desktop & adaptive corrections

taorkon committed:
Merge branch 'main' of https://github.com/taorkon/english-school into dev-Yulia

taorkon committed:
removed margins for socils
